superset/assets/src/SqlLab/components/SaveQuery.jsx

```diff
@@ -108,6 +110,18 @@ class SaveQuery extends React.PureComponent {
           </Col>
         </Row>
         <br />
+        {this.props.saveQueryWarning && (
+          <div>
+            <Row>
```

Comment:
Are the `div->Row->Col` wrappers useful here? Wondering if you could just go straight to a `<small>`

Reply:
I would remove the `<div>` but keep `Row` and `Col` for semantic

Reply:
I need the outer `<div>` to wrap the `<Row>` and `<br />` children. I at least would need:
```
<div>
  <div>
    <small>
      {this.props.saveQueryWarning}
    </small>
  </div>
  <br />
</div>
```